btr-api/src/btr_api/resources/submission.py

```diff
@@ -127,14 +127,26 @@ def create_register():
             # create invoice in pay system
             invoice_resp = btr_pay.create_invoice(account_id, jwt, json_input)
             submission.invoice_id = invoice_resp.json()['id']
-            submission.save()
         except ExternalServiceException as err:
             # Log error and continue to return successfully (does NOT block the submission)
             # TODO: save this information to a table so that a daily job can pick these up and retry them
```

Comment:
Is this planned, can the ticket be created.

Reply:
This is more of an idea I guess and not sure what the specifics will be yet because we are still figuring out the integration a bit. The todo is more of a placeholder for us to do something to handle this so we don't forget. I think we should wait to make tickets after your RFC because we may be moving this code out entirely